ivy/container/statistical.py

```diff
@@ -101,6 +146,78 @@ def var(
             out=out,
         )
 
+    @staticmethod
+    def static_var(
+        x: ivy.Container,
+        axis: Union[int, Tuple[int]] = None,
+        correction: Union[int, float] = 0.0,
+        keepdims: bool = False,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """ ivy.Container static method variant of ivy.var. 
+        This method simply wraps the function, and so 
+        the docstring for ivy.var also applies to this method
+        with minimal changes.
+
+        Parameters
+        ----------
+        self
+            input array. Should have a floating-point data type.
+        key_chains
+            The key-chains to apply or not apply the method to. 
+            Default is None.
+        to_apply
+            If True, the method will be applied to key_chains,
+            otherwise key_chains will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was 
+            not applied. Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). 
+            Default is False.
+        out
+            optional output, for writing the result to. 
+            It must have a shape that the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+           if the variance was computed over the entire array, 
+           a zero-dimensional array containing the variance; 
+           otherwise, a non-zero-dimensional array containing the 
+           variances. The returned array must have the same data 
+           type as x.
+       
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]),\
```

Comment:
write this as:
`x = ivy.Container(a=ivy.array([0.1, 0.2, 0.9]), \`
there should be space before and after the `\` separator.